gammapy/visualization/utils.py

```diff
@@ -3,6 +3,9 @@
 import matplotlib.pyplot as plt
 from gammapy.maps import MapAxis
 from gammapy.maps.utils import edges_from_lo_hi
+import logging
```

Comment:
Note that logging is not longer needed. This can be removed.

Reply:
good point! done